### DISCUSSION TITLE
nothing happen when i click run test

### DISCUSSION THREAD

**Discussion by aelbore** [vitest-dev/vscode#35] [Q&A] [answered]
Any idea why nothing happen happen when i click the run test? thanks is there anything i need to install?

I think i have this issue
Activating extension 'ZixuanChen.vitest-explorer' failed: Cannot get vitest version from "{"cmd":"<my-workspaceFolder>/node_modules/.bin/vitest

**Comment by zxch3n** [comment, marked as answer]
What's your OS? Does [this project](https://github.com/zxch3n/vitest-ext-basic-example) work on your machine?

**Reply by aelbore** [reply]
MacOS 12.3.1 (M1), and i have that files on my machine, vitest version is 0.12.4, i tried the release and pre-release same issue. 
by the way thanks for this plugin

**Reply by aelbore** [reply]
https://user-images.githubusercontent.com/18069807/169027171-b470e932-b4b7-41c8-ba76-0e03e6ad8630.mov

**Reply by zxch3n** [reply]
My pleasure ;) Can you try set `vitest.nodeEnv` as `{"PATH": "/path/to/node/bin"}` in `.vscode/settings.json`?

**Reply by aelbore** [reply]
i tried this, nothing happen :( 

```json
{
  "vitest.enable": true,
  "vitest.nodeEnv": {
    "PATH": "/Users/dev/.nvm/versions/node/v16.13.1/bin/node"
  }
}
```

**Reply by zxch3n** [reply]
I just published a new version v0.2.8 to pre-release. I fixed an issue that may have caused your problem and add a few more log about it. You can try it later with the same settings

**Reply by aelbore** [reply]
@zxch3n thanks for the help, i can now run the tests but now it has new error or issue. maybe i will create a issue and add some screenshot and steps to reproduce.  again thanks for the help

**Reply by aelbore** [reply]
@zxch3n i saw you have new version for pre-release, and my issue is gone thanks. really nice to use this plugin
